dotcom-rendering/src/components/MatchNav.tsx

```diff
@@ -38,36 +38,6 @@ const CrestRow = ({ children }: { children: React.ReactNode }) => (
 	</div>
 );
 
-const StretchBackground = ({ children }: { children: React.ReactNode }) => (
-	<div
-		css={css`
-			display: flex;
-			flex-direction: column;
-			justify-content: space-between;
-			position: relative;
-			padding: ${space[2]}px;
-			background-color: var(--match-nav-background);
-			margin-bottom: 10px;
-			${until.tablet} {
-				margin: 0 -10px 10px;
-			}
-
-			:before {
```

Comment:
We found this wasn't actually taking effect when used in the actual article layout